CONTRIBUTING.md

```diff
@@ -221,21 +221,24 @@ There are two ways to do this.
 1.  **With Nix**
 
     Enter the command `nix build .#mkdocs` (or `nix-build -A mkdocs`) then open the
-    file `result/site/index.html` in a browser.
+    file `result/site/index.html` in a browser. This method always type-checks the
+    Agda code, and generates the HTML documentation from documentation from scratch.
 
     **Note**. This currently works in Chrome but may not work in Brave or
     Firefox.  If you want to use one of those browsers to view the generated
     documentation, you can run a local server on the result,
-    `cd result/site; python3 -m http.server`, and then point your browser
+    `(cd result/site; python3 -m http.server)`, and then point your browser
```

Comment:
I don't think you need commas *and* parentheses here.  I prefer commas, but if you prefer parentheses, then please remove the commas.

Reply:
The parenthesis are not stylistic, they keep you at the root folder when the command returns. But I'm fine undoing that anyway.